fe2-android/app/src/test/java/com/github/dedis/popstellar/model/objects/TransactionObjectTest.java

```diff
@@ -122,9 +122,23 @@ public void getReceiversHashTransactionTest() {
         Collections.singletonList(pubKeyHash), transactionObject.getReceiversHashTransaction());
   }
 
-
+  // test thrown null List<PublicKey> getReceiversTransaction(Map<String, PublicKey> mapHashKey)
+  // test thrown null Map<PublicKey, Long> getReceiversTransactionMap(Map<String, PublicKey>
+  // mapHashKey)
   @Test
   public void getReceiversTransactionTestNull() {
+    /*public List<PublicKey> getReceiversTransaction(Map<String, PublicKey> mapHashKey) {
+      Iterator<String> receiverHashIte = getReceiversHashTransaction().iterator();
+      List<PublicKey> receivers = new ArrayList<>();
+      while (receiverHashIte.hasNext()){
+        PublicKey pub = mapHashKey.getOrDefault(receiverHashIte.next(),null);
+        if (pub == null) {
+          throw new IllegalArgumentException("The hash correspond to no key in the dictionary");
+        }
+        receivers.add(pub);
+      }
+      return receivers;
+    }*/
```

Comment:
If the whole test is commented should you remove it? Or did you just forget to de-comment it